1.0/docs/devguide/registering-elements.md

```diff
@@ -91,6 +91,12 @@ Two notes about the custom constructor:
 
 ### Extend native HTML elements {#type-extension}
 
+**Note:** When using Shadow DOM, extension of native elements can have
+unexpected behavior and is sometimes not permitted. It is suggested that
+you test your element with Shadow DOM enabled to catch any problems
```

Comment:
It is suggested that you test => Test

Shadow DOM enabled => native shadow DOM enabled.

Also, because a lot of people miss the fact that there's a setting, please add a link to the settings page at the end of the note:

    For information on enabling native shadow DOM, see [Global Polymer settings](https://www.polymer-project.org/1.0/docs/devguide/settings.html).